pyproject.toml

```diff
@@ -60,6 +60,7 @@ optional-dependencies.testing = [
   "pytest-mock>=3.10",
   "pytest-xdist>=3.1",
   "re-assert>=1.1",
+  "wheel",
```

Comment:
`wheel>=0.38.4` please

Reply:
I don't understand this requirement. It works fine with wheel 0.37.1 as well.

Reply:
yet we only test in CI with 0.38.4, so 0.37.1 is happenstance not actually validated

Reply:
I see. Wouldn't that be the same issue the minute 0.38.5 is released?

Reply:
Anyway, amended.

Reply:
That explains why we need to patch the requirement versions out so often in Fedora.

Reply:
You can do that, but I'm washing my hands if people submit bugs for it 😆

Reply:
Absolutely undestood.